draft-ietf-quic-transport.md

```diff
@@ -649,85 +757,27 @@ Note:
   the packet is that the client might have received the token in a previous
   connection using the NEW_TOKEN frame, and if the server has lost state, it
   might be unable to validate the token at all, leading to connection failure if
-  the packet is discarded.
+  the packet is discarded.  A server might encode tokens that it provides with
+  NEW_TOKEN frames and Retry packets differently, and validate the latter more
+  strictly.
 
 
-#### Starting Packet Numbers
+### Starting Packet Numbers
 
-The first Initial packet contains a packet number of 0. Each packet sent after
-the Initial packet is associated with a packet number space and its packet
-number increases monotonically in that space (see {{packet-numbers}}).
+The first Initial packet sent by either endpoint contains a packet number of
```

Comment:
SHOULD contain?

Reply:
Or MUST contain?

Reply:
Certainly not MUST. Consider the case where the Initial packet is resent after a version negotiation, or repeated after time-out. Then the "natural" value of the sequence number will be 1. I don't see how saying "must be zero" would improve interoperability.

Reply:
Christian got it.  MUST doesn't work, and throwing normative language at this is too heavy.  I think that this is fine.